webapp/web/dashboard.css

```
@@ -1,91 +1,233 @@
 .dashboard {
-    width: 50%;
-    margin: 40px 30px;
+    margin: 40px 150px;
 }
 
-.dashboard__project-actions {
+.project-actions {
     display: flex;
-    flex-direction: row;
-    justify-content: space-between;
+    flex-flow: row nowrap;
     text-align: center;
-    line-height: 60px;
-    width: 80%;
 }
 
-.dashboard__project-actions:hover {
+.project-actions__action {
+  display: flex;
+  justify-content: space-between;
+  flex-basis: 50%;
+  border: 1px solid #499398;
+  border-radius: 5px;
+  padding: 10px;
+  margin-right: 20px;
+  height: 25px;
+}
+
+.project-actions__action:last-child {
+  margin-right: 0px;
+}
+
+.project-actions__action:hover {
     cursor: pointer;
 }
 
-.dashboard__project-actions-link {
-    color: #000000;
-    text-decoration: none;
-    vertical-align: middle;
-    border-style: solid;
-    border-width: .125rem;
-    padding: 0px 10px;
-    width: 175px;
+.project-actions__action-link {
+  align-self: center;
+  color: #499398;
+  text-decoration: none;
+  font-size: 1.2em;
+  font-weight: 400;
+}
+
+.project-actions__action-icon {
+  align-self: center;
+  color: #499398;
+  font-size: 2em;
+}
+
+.project-summary {
+  margin-top: 50px;
+}
+
+.project-summary__chart {
+  width: 100%;
+}
+
+.project-audience {}
+
+.project-audience-header {
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+}
+
+.project-audience-actions {
+  display: flex;
+  flex-wrap: nowrap;
+  align-self: center;
+}
+
+.project-audience-action {
+  display: flex;
+  flex-wrap: nowrap;
+  margin-left: 20px;
+  font-weight: 400;
+  color: #499398;
 }
 
-.active-packages {}
+.project-audience-action:hover {
+  cursor: pointer;
+}
+
+.project-audience-action__text {
+  white-space: nowrap;
+  font-size: 1.2em;
+}
+
+.project-audience-action__icon {
+  margin-left: 10px;
+  font-size: 1.4em;
+  line-height: 25px;
+}
+
+.project-audience-content {
+  display: flex;
+  flex-direction: column;
+  border: 2px solid #cacaca;
+  padding: 20px;
+}
+
+.project-audience-content-item {
+  display: flex;
+  flex-wrap: wrap;
+  font-size: 1.3em;
+  margin-bottom: 20px;
+}
 
-.active-packages__package {
-    border-style: solid;
-    border-width: 1px;
-    border-radius: .5rem;
-    border-color: rgba(0, 0, 0, .1);
-    padding: 6px;
-    margin-bottom: 20px;
-    width: 80%
+.project-audience-content-item__text {
+  flex-basis: 100%;
```

There's a little bit too much whitespace around these messages, as this element is inheriting the margin of `.project-audience-content-item`. Let's set `margin: 0` here
```suggestion
  flex-basis: 100%;
  margin: 0;
```